recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "1.24.1" %}
+{% set version = "1.25.0" %}
 
 package:
   name: pytest-xdist
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pytest-xdist/pytest-xdist-{{ version }}.tar.gz
-  sha256: 909bb938bdb21e68a28a8d58c16a112b30da088407b678633efb01067e3923de
+  sha256: 96f893094c89fddeaff3f4783f4807f7aeb138be1a0d87a8805057b6af1201b5
 
 build:
   number: 0
   noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
```

Comment:
@nicoddemus I'm not sure what I going on (I'm out of the loop lately due to the day job).
All I know is that reverting to the old `setup.py` "works." Maybe @msarahan knows why this was failing.